arestor/api/__init__.py

```diff
@@ -45,6 +45,6 @@ def config(cls):
                 'server.thread_pool': CONFIG.api.thread_pool,
```

Comment:
being an API change, this should have a commit message that presents the changes